src/screens/Calendar/CalendarScreen.tsx

```
@@ -11,9 +11,60 @@ import { calendarScreenStyles as styles } from "./CalendarScreen.styles";
+  useEffect(() => {
+    const unsubCal = subscribeCalendarDates(() => setCalendarVersion((v) => v + 1));
+    const unsubOotd = subscribeOotdList(() => setCalendarVersion((v) => v + 1));
+    return () => {
+      unsubCal();
+      unsubOotd();
+    };
+  }, []);
+
+  const mergedOotdListData = useMemo(
+    () => ({ ...getCalendarSavedEntries(), ...ootdListData }),
+    [ootdListData, calendarVersion]
```

ESLint

[error] React Hook useMemo has an unnecessary dependency: 'calendarVersion'. Either exclude it or remove the dependency array.

(react-hooks/exhaustive-deps)